### ISSUE TITLE
Sketcher → Sketcher geometries (toolbar) → Create arc

### ISSUE THREAD

**Issue by kaktusus** [FreeCAD/FreeCAD-translations#304]
```
OS: Debian GNU/Linux trixie/sid (KDE/plasma)
Word size of FreeCAD: 64-bit
Version: 0.22.0dev.36228 (Git) AppImage
Build type: Release
Branch: main
Hash: 6c8ee5964521a99e01c8d33cbf9aac8a19273ef3
Python 3.10.13, Qt 5.15.8, Coin 4.0.2, Vtk 9.2.6, OCC 7.6.3
Locale: Polish/Poland (pl_PL)
Installed mods: 

```

Label translations and tooltips are missing.

![obraz](https://github.com/FreeCAD/FreeCAD-translations/assets/39754351/c07ea235-1536-48af-ad07-1acd5896a5f3)

`Center and endpoints` - string is in Crowdin.
    https://crowdin.com/editor/freecad/569/en-pl?view=comfortable#6634598

`Create an arc by its center and by its endpoints` - string is in Crowdin.
    https://crowdin.com/editor/freecad/569/en-pl?view=comfortable#6634860

![obraz](https://github.com/FreeCAD/FreeCAD-translations/assets/39754351/3166ce46-01fc-4758-9a5f-65f1d84396b4)

`Endpoints and rim point` - string is in Crowdin.
    https://crowdin.com/editor/freecad/569/en-pl?view=comfortable#6634602

`Create an arc by its endpoints and a point along the arc` - string is in Crowdin.
    https://crowdin.com/editor/freecad/569/en-pl?view=comfortable#6634862

**Comment by chennes** [comment]
I cannot reproduce -- this is translated for me.

**Comment by kwahoo2** [comment]
I can reproduce.
![obraz](https://github.com/FreeCAD/FreeCAD-translations/assets/3474114/145ca7d6-23f0-4248-bb3f-44fe2c16570d)

```
OS: Fedora Linux 39 (KDE Plasma) (KDE/plasma)
Word size of FreeCAD: 64-bit
Version: 0.22.0dev.36277 (Git) AppImage
Build type: Release
Branch: main
Hash: 9e1903d46112b3660bf10c6a4537d728101d560b
Python 3.10.13, Qt 5.15.8, Coin 4.0.2, Vtk 9.2.6, OCC 7.6.3
Locale: Polish/Poland (pl_PL)
Installed mods: 
  * freecad.gears 1.2.0

```

It seems to be a regression, since in: 0.22.0dev.35966 (Git) AppImage strings were shown as translated.

**Comment by chennes** [comment]
This was probably fixed by [1e164f24c](https://github.com/FreeCAD/FreeCAD/commit/1e164f24c0673976b892d6bba706b98cfc4fea14) then. Can you please check again when the next weekly gets built?

**Comment by kaktusus** [comment]
fixed:

```
OS: Debian GNU/Linux trixie/sid (KDE/plasma)
Word size of FreeCAD: 64-bit
Version: 0.22.0dev.36331 (Git) AppImage
```


![obraz](https://github.com/FreeCAD/FreeCAD-translations/assets/39754351/0549a975-3c5f-45da-b230-2cb0007d2001)
